process/economics/gasConfigHandler.go

```diff
@@ -259,29 +261,30 @@ func checkAndParseGasLimitSettings(gasLimitSetting config.GasLimitSetting) (*gas
 	return gc, nil
 }
 
-func convertGenericValues(economics *config.EconomicsConfig) (uint64, uint64, *big.Int, uint64, error) {
+func convertGenericValues(economics *config.EconomicsConfig) (uint64, uint64, *big.Int, uint64, uint64, error) {
```

Comment:
can you update to return a struct pointer as response and an error?
we have multiple uint64 in return values, and it will be easy to mix them on caller.

Reply:
updated